src/kernel/filesystem/fat32.zig

```diff
@@ -3985,18 +3983,7 @@ test "Fat32FS.openImpl - match short name" {
     var test_fs = try initialiseFAT32(std.testing.allocator, test_fat32_image);
     defer test_fs.destroy() catch unreachable;
 
-    const file_node = try Fat32FS(@TypeOf(test_fat32_image)).openImpl(test_fs.fs, &test_fs.root_node.node.Dir, "INSANE~1.TXT");
-    defer file_node.File.close();
-}
-
-test "Fat32FS.openImpl - match long name" {
-    const test_fat32_image = try std.fs.cwd().openFile("test/fat32/test_fat32.img", .{});
-    defer test_fat32_image.close();
-
-    var test_fs = try initialiseFAT32(std.testing.allocator, test_fat32_image);
-    defer test_fs.destroy() catch unreachable;
-
-    const file_node = try Fat32FS(@TypeOf(test_fat32_image)).openImpl(test_fs.fs, &test_fs.root_node.node.Dir, "insanely_long_insanely_long_insanely_long_insanely_long_insanely_long_insanely_long_insanely_long_insanely_long_insanely_long_insanely_long_insanely_long_insanely_long_insanely_long_insanely_long_insanely_long_insanely_long.txt");
```

Comment:
Could this test be skipped instead of removed, as in the future, this could be fixed.

Reply:
Agreed. How do I do that? Just comment it out or is there a fancy way of skipping tests?

Reply:
Can return `error.SkipZigTest`. This can be found in `special\test_runner.zig`

Reply:
NIce